opal/group.go

```diff
@@ -228,14 +262,11 @@ func resourceGroupCreate(ctx context.Context, d *schema.ResourceData, m any) dia
 	recommendedDurationI, recommendedDurationOk := d.GetOk("recommended_duration")
 	requestTemplateIDI, requestTemplateIDOk := d.GetOk("request_template_id")
 	isRequestableI, isRequestableOk := d.GetOkExists("is_requestable")
-	if adminOwnerIDOk || requireManagerApprovalOk || autoApprovalOk || requireMfaToApproveOk || requireMfaToRequestOk || requireSupportTicketOk || maxDurationOk || requestTemplateIDOk || isRequestableOk {
+	if adminOwnerIDOk || autoApprovalOk || requireMfaToApproveOk || requireMfaToRequestOk || requireSupportTicketOk || maxDurationOk || requestTemplateIDOk || isRequestableOk {
```

Comment:
not important for this review, but what are your thoughts about making this more readable as:
```go
func Or(conditions ...bool) bool {
	for _, condition := range conditions {
		if condition {
			return true
		}
	}
	return false
}

if Or(
	adminOwnerIDOk,
	autoApprovalOk,
	requireMfaToApproveOk,
	requireMfaToRequestOk,
	requireSupportTicketOk,
	maxDurationOk,
	requestTemplateIDOk,
	isRequestableOk,
) {
	...
}
```
it would make diffs a little more readable as well